public/stylesheets/app.css

```diff
@@ -169,3 +169,9 @@ body.dark-mode main a {
 	right: 0.2rem;
 	top: 0;
 }
+
+.navbar-dropdown {
+	background-color: black;
+	padding-top: 0;
+	border-top: 0;
```

Comment:
With `border-top` set the width difference between `scanning` item and dropdown section is annoying.
Without it being set, it's hard to notice the difference (imo).
If you want it to be fixed anyway, just let me know.

Reply:
I tested locally and it works fine. The font for the sub-menu seems a tad small, but we can fix that later.